tests/foreman/api/test_docker.py

```diff
@@ -1489,33 +1490,48 @@ class DockerRegistriesTestCase(APITestCase):
 
     """
 
-    @stubbed()
     @run_only_on('sat')
-    def test_create_registry(self):
+    @data(*valid_data_list())
+    def test_create_registry(self, name):
         """@Test: Create an external docker registry
 
         @Feature: Docker
 
-        @Assert: the external registry is created
-
-        @Status: Manual
+        @Assert: External registry is created successfully
 
         """
+        url = gen_url(subdomain=gen_string('alpha'))
+        description = gen_string('alphanumeric')
+        registry = entities.Registry(
+            name=name,
+            url=url,
+            description=description,
+        ).create()
+        self.assertEqual(registry.name, name)
+        self.assertEqual(registry.url, url)
+        self.assertEqual(registry.description, description)
 
-    @stubbed()
     @run_only_on('sat')
-    def test_update_registry_name(self):
+    @data(*valid_data_list())
+    def test_update_registry_name(self, new_name):
         """@Test: Create an external docker registry and update its name
 
         @Feature: Docker
 
         @Assert: the external registry is updated with the new name
 
-        @Status: Manual
-
         """
+        name = gen_string('alpha')
+        registry = entities.Registry(
+            name=name,
+            url=gen_url(subdomain=gen_string('alpha')),
+            description=gen_string('alpha'),
+        ).create()
+        self.assertEqual(registry.name, name)
+        registry.name = new_name
+        registry = registry.update()
+        self.assertEqual(registry.name, new_name)
```

Comment:
Is there any particular reason to generate an "alpha" description, url or name? If not, the test can be simplified like so:

```python
registry = entities.Registry().create()
registry.name = new_name
self.assertEqual(registry.name, registry.update().name))
```

This works because `update` takes the server's response and creates a brand new object instead of modifying the object it's called on.